examples/recip_compressor.py

```diff
@@ -79,33 +79,38 @@ def Compressor():
     C_D = 1.17              #Drag coefficient [-]
     d_valve = 0.007         #Valve Diameter [m]
     x_stopper = 0.0018      #Stopper location [m]
-    
+
+    I=(d_valve*h_valve**3)/12  #Moment of Intertia for valve,[m^4]
+    k_valve=(6*E*I)/(a_valve**2*(3*l_valve-a_valve))    #Valve stiffness
+    m_eff=(1/3)*rho_valve*l_valve*d_valve*h_valve      #Effective mass of valve reeds
+    x_tr_suction = 0.25*(recip.d_suction**2/d_valve)
+    x_tr_discharge = 0.25*(recip.d_discharge**2/d_valve)
+
     #The suction valve parameters
```

Comment:
@ibell : this is how it is done for a simple recip example. Alternatively I can create a function inside recip/core.py
It is quite straight forward. I just took them out from ValveClass. The example works nicely.